src/lib.rs

```diff
@@ -228,9 +228,14 @@ pub struct Config {
     /// request, which most times is 50.
     pub pagination_chunks: u32,
 
+    /// Whether or not to save the authentication token into a JSON file,
+    /// then reread token from JSON file when launch the program without
```

Comment:
typos: 

* "then reread token" -> "then reread the token"
* "when launch the program" -> "when launching the program"